vllm_ascend/worker/model_runner_v1.py

```diff
@@ -193,10 +195,7 @@ def __init__(self, vllm_config: VllmConfig, device: torch.device):
                 device=self.device)
 
         self.graph_block_tables = np.zeros(
-            (self.max_num_reqs,
-             (self.model_config.max_model_len + self.block_size - 1) //
```

Comment:
`self.max_num_blocks_per_req`  is equal to this value, no need to compute again.